python-requirements.txt

```diff
@@ -6,15 +6,16 @@
 beautifulsoup4==4.12.2
 hjson==3.1.0
 jsonschema==4.17.3; python_version >= "3.7"
+libclang==16.0.0
```

Comment:
@timothytrippel Any concerns re airgapped builds?

Reply:
yes it seems likely this will cause an issue, will need to test it to make sure, but I think we will want to hold off on merging this for a couple weeks.

Reply:
Resolving this since @timothytrippel [confirmed that this doesn't cause any issues](https://github.com/lowRISC/opentitan/pull/18719#issuecomment-1639061617).